tick/optim/prox/src/prox_multi.cpp

```diff
@@ -1,43 +1,38 @@
-//
-// Created by Stéphane GAIFFAS on 30/12/2015.
-//
-
 #include "prox_multi.h"
 
-
 ProxMulti::ProxMulti(std::vector<ProxPtr> proxs,
                      double strength)
-    : Prox(strength), proxs(proxs) {
-    if (strength != 0) {
-        throw std::invalid_argument("ProxMulti can be instantiated with strength=0 only, since "
-                                            "ProxMulti's strength is not used");
-    }
+  : Prox(strength, false), proxs(proxs) {
+  if (strength != 0) {
```

Comment:
We shouldn't do equality comparison with floating points. If strength is a computed value it will break (but I guess it will be initialized to 0.0 all the time)